packages/link-bins/src/index.ts

```diff
@@ -120,6 +120,10 @@ async function getPackageBins (
     return []
   }
 
+  if (typeof pkg.bin === 'string' && !pkg.name) {
+    throw new Error(`Package in ${target} must have a name to get bin linked.`)
```

Comment:
Use the PnpmError with an error code. In that case pnpm will print a nice error message with no stacktrace

Reply:
Will do. For future implementations, do we always prefer to use PnpmError?

Reply:
yes